pkg/kubectl/resource_printer.go

```diff
@@ -220,15 +224,15 @@ func (h *HumanReadablePrinter) validatePrintHandlerFunc(printFunc reflect.Value)
 		return fmt.Errorf("invalid print handler. %#v is not a function", printFunc)
 	}
 	funcType := printFunc.Type()
-	if funcType.NumIn() != 4 || funcType.NumOut() != 1 {
+	if funcType.NumIn() != 5 || funcType.NumOut() != 1 {
 		return fmt.Errorf("invalid print handler." +
-			"Must accept 3 parameters and return 1 value.")
+			"Must accept 5 parameters and return 1 value.")
```

Comment:
Refactor in another PR sgtm. Updating the comment to be accurate is fine for now.